src/collections/_documentation/platforms/python/django.md

```diff
@@ -39,6 +39,29 @@ This causes the following this to happen:
 * Logging with any logger will create breadcrumbs when the [Logging]({% link _documentation/platforms/python/logging.md %})
   integration is enabled (done by default).
 
+## Options
+
+The following keyword arguments can be passed to `DjangoIntegration()`:
+
+* `transaction_style`:
+
+  ```python
+  @app.route("/myurl/<foo>")
+  def my_function(request):
```

Comment:
This looks like [Flask code](flask.pocoo.org/docs/latest/quickstart/). Are you sure this is meant for the Django-related documentation?

If it is the code sample probably needs some explanation. Explain why the annotation is needed and what it does.

Reply:
The decorator is a copypaste error, yes. Thanks for bringing this to our attention.

Reply:
https://github.com/getsentry/sentry-docs/pull/623